fission/function.go

```diff
@@ -654,6 +654,8 @@ func fnLogs(c *cli.Context) error {
 func fnPods(c *cli.Context) error {
 	client := getClient(c.GlobalString("server"))
 
+	fmt.Println("This subcommand is deprecated and will be remove in the next release. Please use `kubectl -n <namespace> logs -f -c <container> <pod>` instead.")
```

Comment:
will be removed*

The command actually prints out pods names, so the replacement would be something like `kubectl get pods -l functionName=xyz --all-namespaces`